runtime/src/iree/base/internal/dynamic_library_win32.c

```diff
@@ -63,13 +63,13 @@ static void iree_dynamic_library_init_temp_paths(void) {
   // See:
   // https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-gettemppatha
   char temp_path[MAX_PATH];
-  DWORD temp_path_length = GetTempPathA(IREE_ARRAYSIZE(temp_path), temp_path);
+  GetTempPathA(IREE_ARRAYSIZE(temp_path), temp_path);
```

Comment:
`GetTempPathA` returns 0 if creation fails, but this method does not return a value... so we can't do much to handle that besides crash.

https://learn.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-gettemppatha